docs/src/content/docs/guides/make-onetime-payment.mdx

```diff
@@ -140,11 +140,11 @@ Create an Interactive Grant guide. </LinkOut>
 
 ### 8. Create an Outgoing Payment
 
-Create an `outgoingPayment` on the payer wallet's RS using the access token returned by the AS in the grant request.
+Create an `outgoingPayment` on the sender's wallet's RS using the access token returned by the AS in the grant request.
```

Comment:
Maybe "...on the sender wallet's RS" like you have previously (so make sender singular).